web/views.py

```
@@ -4700,6 +4702,286 @@
+
+    return redirect("student_management", course_slug=course.slug, student_id=enrollment.student.id)
+
+
+@login_required
+@teacher_required
+@require_POST
+def award_badge(request):
+    """
+    AJAX view for awarding badges to students.
+    """
+    if not request.headers.get("X-Requested-With") == "XMLHttpRequest":
```

Ruff (0.8.2)

Use `request.headers.get("X-Requested-With") != "XMLHttpRequest"` instead of `not request.headers.get("X-Requested-With") == "XMLHttpRequest"`

Replace with `!=` operator

(SIM201)